Formula/seqkit.rb

```diff
@@ -4,22 +4,13 @@ class Seqkit < Formula
   homepage "https://bioinf.shenwei.me/seqkit/"
   # We use binaries to avoid compiling Go code
   if OS.mac?
-    url "https://github.com/shenwei356/seqkit/releases/download/v0.7.2/seqkit_darwin_amd64.tar.gz"
-    sha256 "6b34e400e9d0437516743b18ffa57b940f2f66ca98b40f4c52562fc2a7ce50f9"
+    url "https://github.com/shenwei356/seqkit/releases/download/v0.9.0/seqkit_darwin_amd64.tar.gz"
+    sha256 "0b29fe3921002f098d134c1a5d2c07b08fe560df966d02af723cabffecc4ab03"
   else
-    url "https://github.com/shenwei356/seqkit/releases/download/v0.7.2/seqkit_linux_amd64.tar.gz"
-    sha256 "de75151fc93ee24717370f9d54aff2f76a04be14ae2dd9384d749b060c210124"
+    url "https://github.com/shenwei356/seqkit/releases/download/v0.9.0/seqkit_linux_amd64.tar.gz"
+    sha256 "56b01963ad272cd388fff25d92115fd7b301c8faef79322ade1d036cfd437781"
   end
-  version "0.7.2"
-
-  bottle do
-    root_url "https://linuxbrew.bintray.com/bottles-bio"
-    cellar :any_skip_relocation
-    sha256 "7a69ce7a8b1c53858005754c4f3725262a0539166c3fcc316f2ac25f422f0363" => :sierra_or_later
-    sha256 "c17453614853dcb7bf49d94c3f395f9b4fb2c162c4667adfe0d73b0988621c82" => :x86_64_linux
-  end
```

Comment:
For future PRs, there's no need to remove the `bottle` block. You can leave it as is.